templates/userdash/userdash-detail.html

```diff
@@ -0,0 +1,81 @@
+{{ define "content" }}
+
+{{ template "basicspotlight" . }}
+
+<section class="container">
+	<p class="lead text-center">You can find details of your order here.</p>
+	<div class="row justify-content-center mt-5">
+		<div class="col-md-8 col-lg-6">
+			{{ if .error }}
+			<div class="alert alert-danger text-center" role="alert">
+				{{ rawHTML .error }}
+			</div>
+			{{ end }}
+			<hr />
+			<div class="d-flex justify-content-between align-items-center">
+				<div>
+					<h5 class="my-1">{{ .selectedMealName }}</h5>
+					<p class="text-muted font-italic">Meal Combo</p>
+				</div>
+				<div>
+					<h5><strong>{{formatMoney .selectedMealPrice}}</strong></h5>
+				</div>
+			</div>
+			<div class="d-flex justify-content-between align-items-center">
+				<div>
+					<h5 class="my-1">{{ .selectedDrinkName }}</h5>
+					<p class="text-muted font-italic">Drink</p>
+				</div>
+				<div>
+					<h5 class="text-muted">included</h5>
+				</div>
+			</div>
+			<div class="d-flex justify-content-between align-items-center">
+				<div>
+					<h5 class="my-1">Delivery Fee</h5>
+				</div>
+				<div>
+					<h5 class="font-weight-bold">{{formatMoney .deliveryFee}}</h5>
+				</div>
+			</div>
+			<hr/>
+			<div class="d-flex justify-content-between align-items-center">
+				<div>
+					<h5 class="my-1 font-weight-bold text-ca-yellow2">Total</h5>
+				</div>
+				<div>
+					<h5 class="font-weight-bold text-ca-yellow2">{{formatMoney .orderTotal}}</h5>
+				</div>
+			</div>
+			<hr/>
+			<h4 class="text-center">Payment</h4>
+			<div class="d-flex justify-content-between align-items-center">
+				<div>
+					<h5 class="my-1">Your Caf Account</h5>
+				</div>
+				<div>
+					<h5 class="font-weight-bold text-danger">{{formatMoney .cafAccountChargeAmount}}</h5>
+				</div>
+			</div>
+			<div class="d-flex justify-content-between align-items-center">
+				<div>
+					<h5 class="my-1">CafApp Balance</h5>
+				</div>
+				<div>
+					<h5 class="font-weight-bold text-danger">{{formatMoney .deliveryFee}}</h5>
+				</div>
+			</div>
+			<hr />
+			{{ if .isDelivered }}
+				<p class="text-center">Order was delivered to: {{ .destination }} at {{ .CreatedAt.Format "Jan _2 3:04pm" }}</p>
+			{{ else }}
```

Comment:
@dpham273 Have you actually test this out? You can make it become delivered by simply changing the status code in the DB. Should be able to do that using the mysql workbench thing. Pls try it because I suspect this line is wrong